packages/eslint-plugin/package.json

```diff
@@ -38,19 +39,24 @@
     "accessibility"
   ],
   "dependencies": {
-    "@eslint/plugin-kit": "0.2.8",
+    "@eslint/plugin-kit": "0.3.1",
     "@html-eslint/parser": "^0.40.0",
     "@html-eslint/template-parser": "^0.40.0",
     "@html-eslint/template-syntax-parser": "^0.40.0"
   },
+  "peerDependencies": {
+    "eslint": "^9.7.0"
+  },
   "devDependencies": {
-    "@eslint/core": "0.13.0",
+    "@eslint/core": "0.14.0",
     "@html-eslint/types": "^0.40.0",
-    "@types/eslint": "^9.6.1",
     "@types/estree": "^0.0.47",
     "es-html-parser": "0.2.0",
     "eslint": "^9.21.0",
     "espree": "^10.3.0",
-    "typescript": "^5.7.2"
+    "typescript": "^5.8.3"
+  },
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Comment:
This is version ESlint uses (c.f. `@eslint/plugin-kit`)

Reply:
While we still support ESLint v8, we depend on `@eslint/plugin-kit` which still requires v18.18+.